src/rules/declaration-block-no-redundant-longhand-properties/README.md

```diff
@@ -0,0 +1,131 @@
+# declaration-block-no-redundant-longhand-properties
+
+Disallow longhand properties that can be combined into one.
+
+```css
+a {
+    padding-top: 1px;
+    padding-right: 1px;
+    padding-bottom: 1px;
+    padding-left: 1px; }
+/** ↑
+ *  These longhand properties */
+```
+
+The longhand properties in the example above can be more concisely written as:
+
+```css
+a {
+  padding: 1px;
+}
+```
+
+This rule warns when the following shorthand properties can be used:
+
+- `padding`
+- `margin`
+- `background`
+- `font`
+- `border`
+- `border-top`
+- `border-bottom`
+- `border-left`
+- `border-right`
+- `border-width`
+- `border-style`
+- `border-color`
+- `border-radius`
+- `transition`
+
```

Comment:
I think `border` is like a *super* shorthand property for other shorthand properties. For example, `border-right` [is a](https://developer.mozilla.org/en/docs/Web/CSS/border-right) shorthand property for the `border-right-color`, `border-right-style`, and `border-right-width` properties.